docs/snakefiles/reporting.rst

```diff
@@ -246,36 +254,77 @@ Further, using ``url_args()`` you can add URL arguments and using ``anchor()`` y
             file=f,
         )
 
+.. _snakefiles-rendering_reports:
+
 Rendering reports
 -----------------
 
-To create the report simply run
+All the metadata contained in the report (e.g. runtime statistics) is automatically collected during the rendering of the report.
+These statistics are obtained from the metadata that is stored in the ``.snakemake`` directory inside your working directory.
+
+.. _snakefiles-self_contained_html_file:
+
+Self-contained HTML file
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create a default report simply run
 
 .. code-block:: bash
 
-    snakemake --report report.html
+    snakemake --report
 
-after your workflow has finished.
-All other information contained in the report (e.g. runtime statistics) is automatically collected during creation.
-These statistics are obtained from the metadata that is stored in the ``.snakemake`` directory inside your working directory.
+after your workflow has finished successfully.
+This will implicitly assume a report file name of ``report.html`` and will create this self-contained HTML report.
```

Comment:
This will create a report file named ``report.html`` which includes a self-contained HTML report.

The words "implicitly assume" are semantically wrong associated with the output, because, the file name is a default and hence, it will "implicitly create". But then, it is explicitly created ... . Perhaps the shortened version above will do?

Reply:
What about:
```suggestion
This will create a self-contained HTML report with the default filename ``report.html``.
```